global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "7.0.100",
+    "version": "8.0.100",
```

Comment:
If changing this in this branch is not desirable, engineering would need to be changed, so that a `global.json` from e.g. `tests/Standalone/BlazorApp` applies when building that solution.